Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang:1.16-alpine as builder
+
+WORKDIR /workspace
+
+RUN apk add --no-cache git
```

Comment:
I'm guessing the `github` package needs `git` installed for the builder image?

Reply:
...or not, I see you removed it 😄

Reply:
Just a copypasta error on my part, removed.